sonic-pcied/scripts/pcied

```diff
@@ -53,6 +54,49 @@ class DaemonPcied(DaemonBase):
         self.state_db = swsssdk.SonicV2Connector(host=REDIS_HOSTIP)
         self.state_db.connect("STATE_DB")
 
+    # Load AER-fields into STATEDB
+    def update_aer_to_statedb(self, device, device_name, pcie_hash_name):
+
+        aer_fields = {}
+        client = redis.Redis(db=STATE_DB_INDEX)
```

Comment:
can you use sonic_py_common.daemon_base api instead of redis?

Reply:
Sure. Will add a commit to use daemon_base, swsscommon methods instead of redis for DB access.